pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -736,6 +742,9 @@ func getServersConfig(cr *api.KieApp) ([]api.ServerTemplate, error) {
 				instanceTemplate.Jvm = *serverSet.Jvm.DeepCopy()
 			}
 
+			// JBPM configuration
+			instanceTemplate.JbpmCluster = serverSet.JbpmCluster
+
```

Comment:
this is unnecessary, we're already setting at line 674

Reply:
removed

Reply:
removed